Tangra 3/VideoOperations/LightCurves/frmLightCurve.cs

```diff
@@ -1551,15 +1551,60 @@ private void miLoad_Click(object sender, EventArgs e)
 
         private void miSaveAsImage_Click(object sender, EventArgs e)
         {
-			if (m_Graph == null) return;
+            if (m_Graph == null) return;
+
             Bitmap bmp = AddImageLegend();
 
-			if (saveImageDialog.ShowDialog(this) == DialogResult.OK)
-			{
+            SetSaveAsImageFormatToLastUsed();
+
+            if (saveImageDialog.ShowDialog(this) == DialogResult.OK)
+            {
+                string file_ext = Path.GetExtension(saveImageDialog.FileName);
```

Comment:
Please use the same naming convention used across the code base. Typically in C# local variables use camel case i.e. `fileExt` in this case. This is also the conversion used in Tangra. Snake case (`file_ext`) is a Python thing as the name suggests :)

Fro the records Tangra also uses a non-standard C++ convention for member variables with an m_CamelCase prefix and s_CamelCase for static ones.